webhooks/tasks.py

```diff
@@ -0,0 +1,71 @@
+"""
+Celery tasks.
+"""
+import hashlib
+import hmac
+import json
+import logging
+
+from django.apps import apps
+from django.utils.encoding import force_bytes
+import requests
+
+from ccxcon.celery import async
+from webhooks.models import Webhook
+
+log = logging.getLogger(__name__)
+
+
+@async.task
+def publish_webhook(model_str, pk):
+    """
+    Publishes model updates to various services as defined by Webhook instances.
+    """
+    try:
+        app, model_label = model_str.split('.')
+    except ValueError:
+        log.warning("Malformed model string for webhook serialization: %s",
+                    model_str)
+        return
+
+    try:
+        model = apps.get_model(app, model_label)
+    except LookupError:
+        log.warning("Could not find the model %s for webhook publishing.",
+                    model_str)
+        return
+
+    try:
+        instance = model.objects.get(pk=pk)
+        if not hasattr(instance, 'to_webhook'):
+            log.info("Not submitting %s:%s to webhook. It does not support "
+                     "serialization", type(instance), pk)
+            return
+
+        payload = {
+            'action': 'update',
```

Comment:
Just to clarify, this is meant to update, or create if the data doesn't already exist?